NEWS

```diff
@@ -4,12 +4,104 @@ Noteworthy changes in release a.b
 Features and Updates
 --------------------
 
-* New method `hts_idx_nseq` returns the number of contigs covered by reads
-  from an index structure.
-
 * In case a PG header line has multiple ID tags supplied by other applications,
   the header API now selects the first one encountered as the identifying tag
   and issues a warning when detecting subsequent ID tags.
+  (#1256; fixed samtools/samtools#1393)
+
+* VCF header reading function (vcf_hdr_read) no longer tries to download a
+  remote index file by default.
+  (#1266; fixes #380)
+
+* Transparently treat FASTQ as unmapped data as if it was SAM, BAM or CRAM.
+  (#1156)
+
+* Added GCP requester pays bucket access.  Thanks to @indraniel.
+  (#1255)
+
+* Made mpileup's overlap removal choose a random sequence.
+  (#1273; fixes samtools/bcftools#1459)
+
+* Now permits platform specific BAQ parameters.  This also selects long-read
+  parameters for read lengths bigger than 1kb.  This helps bcftools mpileup call
+  SNPs on PacBio CCS reads.
+  (#1275)
+
+* Improved bcf_remove_allele_set.  This fixes a bug that stopped iteration over
+  alleles prematurely, marks removed alleles as 'missing' and does automatic
+  lazy unpacking.
+  (#1288; fixes #1259)
+
+* [CRAM] Improved compression metrics for unsorted files.  This improves the
+  choice of codecs when handling unsorted data.
+  (#1291)
+
+* Now initialises the missing entries of the linear index in reverse order.
+  Populates empty linear index entries from the right rather from the left.
+  The new hts_bin_level function computes the index level of a bin.
+  Thanks to @carsonh for reporting the issue.
+  (#1286; fixes #486)
+
+* Related to the above, the new method, hts_idx_nseq, now returns the total
+  number of contigs from an index.
+  (#1295 and #1299)
+
+* Added bracket handling to bcf_hdr_parse_line.  Thanks to Alberto Casas Orrtiz.
```

Comment:
```suggestion
* Added bracket handling to bcf_hdr_parse_line.  Thanks to Alberto Casas Ortiz.
```

Reply:
Good catch.  I'm too heavy on the 'R' key.

Reply:
I will update my latest commit to fix this.

Reply:
(More for the trivial commit: there were a few trailing spaces near the Hulselmans entry too.)